packages/nuxt/src/app/composables/fetch.ts

```diff
@@ -131,7 +143,8 @@ export function useFetch<
     immediate,
     getCachedData,
     deep,
-    watch: watch === false ? [] : [_fetchOptions, _request, ...(watch || [])]
+    watch: watch === false ? [] : [_fetchOptions, _request, ...(watch || [])],
+    _useFetch: true
```

Comment:
One solution to hide it is to use a Symbol

Reply:
Tell me how 😄

Reply:
we could also avoid typing it entirely